src/particles/plasma/PlasmaParticleContainerInit.cpp

```diff
@@ -99,6 +100,11 @@ InitParticles (const amrex::IntVect& a_num_particles_per_cell,
 
         if (m_do_symmetrize) {
             total_num_particles *= 4;
+            scale_fac /= Hipace::m_normalized_units ? 1. : 4.;
+        } else if (reproducible_temperature) {
+            total_num_particles *= 2*reproducible_temperature;
+            scale_fac /= Hipace::m_normalized_units ? 1. :
+                2.*reproducible_temperature;
```

Comment:
I observed that in SI the plasma density was changed (I assume by a factor of 4) when `do_symmetrize = 1`. Is this expected? Is it different in normalised units? These few lines should fix this (and do the same for reproducible_temperature).

Reply:
This was also a bug in normalised units, this PR proposes to fix it.

Reply:
The fix for do_symmetrized was done in separate PR #957.